index.html

```diff
@@ -4,11 +4,10 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
-  %sveltekit.head%
+  <link rel="stylesheet" href="/src/styles.css">
 </head>
 <body>
-  <div id="root" class="contents">
-    %sveltekit.body%
-  </div>
+  <div id="root" class="contents"></div>
```

Comment:
the `class="contents"` is important because it makes sure that the wrapper div is not rendered by the browser engine.